dsc/cli.py

```diff
@@ -105,7 +106,11 @@ def post_main_group_subcommand(
 def reconcile(ctx: click.Context) -> None:
     """Reconcile bitstreams with item identifiers from the metadata."""
     workflow = ctx.obj["workflow"]
-    workflow.reconcile_bitstreams_and_metadata()
+    try:
+        workflow.reconcile_bitstreams_and_metadata()
+    except ReconcileError:
+        logger.info("Reconcile failed.")
+        ctx.exit(1)
```

Comment:
It's difficult to pin down what are good exit code conventions, but my understanding is that `1` is a good "general error" code that is not overly prescriptive.  So, +1 to `1` for the exit code indicating that reconciliation has failed.